routes/api.php

```diff
@@ -196,6 +196,8 @@ function () {
                 Route::prefix('admin')->group(
                     function () {
                         Route::get('/airfields', 'Admin\\StandAdminController@getAirfields');
+                        Route::get('/airfields/{airfield:code}/terminals', 'Admin\\StandAdminController@getTerminals');
```

Comment:
Is it worth adding airfields to a route group to avoid repitition?

Reply:
Used a prefix, good shout.